ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/RESTMusicService.kt

```diff
@@ -607,11 +606,9 @@ open class RESTMusicService(
 
     @Throws(Exception::class)
     override fun getDownloadInputStream(
-        context: Context,
         song: MusicDirectory.Entry,
         offset: Long,
-        maxBitrate: Int,
-        task: CancellableTask
+        maxBitrate: Int
     ): Pair<InputStream, Boolean> {
         val songOffset = if (offset < 0) 0 else offset
```

Comment:
Neither Task nor Context are needed here or used. Removing them gets rid of a minor circular reference (Task->Service->Task)